lib/tasks/projects.rake

```diff
@@ -119,4 +119,18 @@ namespace :projects do
     exit if ENV['READ_ONLY'].present?
     ProjectDependentRepository.refresh
   end
+
+  supported_platforms = ['Maven', 'npm', 'Bower', 'PyPI', 'Rubygems', 'Packagist']
+
+  desc 'Create maintenance stats for projects'
+  task create_maintenance_stats: :environment do
+    exit if ENV['READ_ONLY'].present?
+    Project.no_existing_stats.where(platform: supported_platforms).limit(500).each(&:update_maintenance_stats_async)
+  end
+
+  desc 'Update maintenance stats for projects'
+  task update_maintenance_stats: :environment do
+    exit if ENV['READ_ONLY'].present?
+    Project.least_recently_updated_stats.where(platform: supported_platforms).limit(500).each{|project| project.update_maintenance_stats_async(priority: :low)}
```

Comment:
This enqueues the update worker in low priority.

Reply:
Any reason to not use find_each and just churn through it?

Reply:
That is one option. The way it is currently set up you would not be able to add the same project to a higher priority queue if it is already enqueued at a lower priority. So I think we would want to change that. I figured this rake task would be called from a cron job, potentially every hour or so. We could instead put all projects on the queue and run that every few weeks instead.